Makefile

```diff
@@ -176,7 +176,14 @@ edit-infra-secrets: read-keyvault-config install-fetch-config set-azure-account
 		-e -d azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_INFRA_SECRET_NAME} -f yaml -c
 
 .PHONY: shell
-shell: get-cluster-credentials ## Open a shell on the app instance on AKS, eg: make qa shell
+shell: get-cluster-credentials ## Actually opens a Rails Console: Open a shell on the app instance on AKS, eg: make qa shell
```

Comment:
I see the diff is confusing here - this is actually the existing `shell` command, which is confusing because it does open a Rails Console

Lines 185 and 186 add the same command actions but using `console`. 

After merging both `make shell` and `make console` will be the exact same. 
`make shell` has been left untouched as we don't know how/where this is used

Reply:
Looking good. 

We can remove the first part of the comment now, so it can just be `## Open a shell on the app instance on AKS, eg: make qa shell`

Reply:
Should we update the comment on line 179? The rest looks good to me 🚀